pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -38,6 +40,8 @@ def __init__(
         date_column: str,
         channel_columns: List[str],
         adstock_max_lag: int,
+        time_varying_media_effect: bool = False,
```

Comment:
Hey, small comment here... This is a matter of taste but I think having two parameters for the same component is a bit strange. What do you think if we evaluate with strings or tuples?

```
time_varying = ('intercept', 'media') #or
```

```
time_varying = 'intercept-media'
```

I think it would be better inside for the API

Reply:
I quite like this. Maybe this gets too complicated but:

```Python
time_varying='intercept'
...
time_varying='total_media'
...
time_varying=['intercept', 'total_media']
...
time_varying=['intercept', 'channel1', 'channe2']
```

could work

Reply:
Maybe even 

```Python
time_varying=['intercept', ('channel1', 'channel2')] # now channel1 and channel2 are summed and multiplied by a time varying coef
```